tests/app/views/contexts/test_hub_context.py

```diff
@@ -1,11 +1,41 @@
+import pytest
+
+from app.data_model.answer_store import AnswerStore
 from app.data_model.list_store import ListStore
 from app.data_model.progress_store import ProgressStore, CompletionStatus
-from app.data_model.answer_store import AnswerStore
 from app.questionnaire.questionnaire_schema import QuestionnaireSchema
+from app.questionnaire.router import Router
 from app.views.contexts.hub_context import HubContext
 
 
-def test_get_not_started_row_for_section():
+@pytest.fixture(name='schema')
+def fixture_schema():
```

Comment:
This was to prevent disabling the file from `# pylint: disable=redefined-outer-name` and this was the suggested approach and it used in our other tests. However, I have moved the fixtures into the `conftest.py` to avoid this problem as discussed.

Reply:
Still doesn't fix it so I used `# pylint: disable=redefined-outer-name`